sources/quest_2/chapter_2/step_9/step_9.md

```diff
@@ -0,0 +1,18 @@
+# Struct in Move
+
+In Move, a struct is like a custom type. Think of it as a simple way to store information with names and values. The name is the property, and the value is what's stored.
+
+Syntax:
+```
+struct NAME has Abilities { FIELD1: TYPE1, FIELD2: TYPE2, ... }
+```
+A `struct` can have up to 4 abilities that determine how values of this type can be used:
+- **Copy:** Value can be copied or cloned by value.
+- **Drop:** Value can be dropped at the end of its scope.
+- **Key:** Value can be used as a key for global storage operations.
+- **Store:** Value can be stored inside global storage.
+
+This helps in organizing and managing data in Move programs.
+
```

Comment:
Good explanation, 

We'll have to clarify the notion of "abilities" later in the course, but for now it's perfect. 

Maybe we should just add an example of structure like it can be in the final code to orient the student but on something really basic

We often like to show a little example of code in courses !

Reply:
Sure!